### PR TITLE
feat: Added support for Romanian locale

### PR DESCRIPTION
**Pull request by stefy**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

Added support for Romanian locale

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Added support for Romanian locale         |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/i18n.en-US.md](https://github.com/stefy/ant-design/blob/feature/docs/react/i18n.en-US.md)
[View rendered docs/react/i18n.zh-CN.md](https://github.com/stefy/ant-design/blob/feature/docs/react/i18n.zh-CN.md)

### CODE REVIEW

File: docs/react/i18n.zh-CN.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ return (
 | 泰米尔人         | ta_IN  |
 | 泰语             | th_TH  |
 | 土耳其语         | tr_TR  |
+| 罗马尼亚         | ro_RO  |
```
Comment (afc163) [inline docs/react/i18n.zh-CN.md]: 罗马尼亚语

Comment (stefy) [reply docs/react/i18n.zh-CN.md]: Fixed

**Review by afc163 — DISMISSED** [review]

**Comment by afc163** [comment]
Please add test case like other language and update snapshots.

https://ant.design/docs/react/contributing#Sending-a-Pull-Request

**Comment by stefy** [comment]
> Please add test case like other language and update snapshots.
> 
> https://ant.design/docs/react/contributing#Sending-a-Pull-Request

Added test cases for Romanian locales and updated the snapshots. Thanks!

**Review by afc163 — APPROVED** [review]

**Pull request merged**